src/base/net/impl/net_thread_impl.cc

```diff
@@ -211,7 +223,7 @@ void NetThread::NetThreadImpl::EnqueueDownload_NetThread(
   auto [iter, inserted] =
       active_downloads_.emplace(easy_handle, std::move(download_info));
   DCHECK(inserted);
-  const auto& inserted_info = *iter;
+  const auto& inserted_info = iter->second;
```

Comment:
Comment: This was a bug fixed by this commit. Previously CURL wrote over the `std::pair<Iterator, bool>` returned by `std::map::emplace()` above instead of the actual value object (`DownloadInfo`).